### ISSUE TITLE
Alternative appearance for 'l' or capital 'I'

### ISSUE THREAD

**Issue by Namchee** [sharanda/manrope#60]
Hi,

I stumbled upon this font accidentally and I didn't regret it at all. Manrope is a great font and it's all thanks to you!
However, it seems Manrope has some issues with distinguishable I and l (capital i dan l), which clearly visible with the `Il1` test.

May I suggest that you add alternate l or capital I for better readability?

Cheers!

**Comment by sharanda** [comment]
@Namchee thanks for your comment. Can you give an example where same height /l and /I appears to be unacceptable?

**Comment by Namchee** [comment]
Sure, I'll make an example by typing some few sentences. Here's the markup

```html
p {
  font-family: 'Manrope', sans-serif;
}

<p>I would like 1 little icicle</p>
<p>Ill behaviour is caused by an AI called Al III commissioned by the little men in the Illogical Illuminati</p>
<p>AI and Al Capone</p>   
```

And here's the result (font is served through Google Font and taken from Chrome)

![manrope](https://user-images.githubusercontent.com/32661241/97794633-17024f00-1c2f-11eb-835a-1bd3a0f958a6.png)

As you can see, it's really hard to distinguish capital i and l on those sentences especially between AI (Artificial Intelligence) and Al (**Al**uminium)

**Comment by howdev** [comment]
you need huge font sizes like 20 and above to see difference in height of capital i and lowercase l. When you use in the UI it is 10-12 and they looked exactly the same. So this is really a big issue.

**Comment by sharanda** [comment]
@howdev @Namchee in the next update, I'll add a taller /l as contextual alternative. Means, it will appear taller when it stands before or after capital /i. In other scenarios it'd appear the same.

**Comment by sharanda** [comment]
Fixed in v 4.505 (will publish soon)
